harvester_e2e_tests/integrations/test_4_vm_snapshot.py

```diff
@@ -315,6 +315,14 @@ def action(sh):
         assert 200 == code
         assert data.get("status", {}).get("readyToUse") is True
 
+    @pytest.mark.dependency(depends=["source_vm_snapshot"])
+    def test_volume_snapshot_not_exist(self, api_client):
+        ''' ref: https://github.com/harvester/tests/issues/524 '''
+        code, data = api_client.vol_snapshots.get()
+
+        assert 200 == code, (code, data)
+        assert not data['data'], (code, data)
+
```

Comment:
A test run _harvester-install-and-test#60_ looks good except this case.
Please help to check, thx.

* Environment: _harvester-v1.3.1_
* Parameter: `test_1_volumes.py or test_4_vm_snapshot.py`

![image](https://github.com/user-attachments/assets/10faf620-b1b4-47ba-a15c-ce2d5c86555a)